src/platforms/php/common/integrations.mdx

```diff
@@ -8,49 +8,42 @@ redirect_from:
 
 ## Default Integrations
 
-Default integrations are integrations enabled by default that integrate into the
-standard library or the interpreter itself. They are documented so you can see
-what they do and that they can be disabled if they cause issues. To disable
-system integrations set `default_integrations => false` when calling `init()`.
+These integrations are enabled by default and integrate into the
+standard library or the interpreter itself. They are documented so you can be both aware of what they do and disable them if they cause issues.
+
+To disable system integrations, set `default_integrations => false` when calling `init()`.
```

Comment:
Actually, since in PHP we have namespaces the `init()` function resides under the `Sentry` namespace. Not sure it's worth mentioning it here, but I see that our examples are always explicit to avoid people copy-pasting to have errors so I will let you decide what to do. Should you decide to add the namespace to the snippet, remember to use the backslash as first character so that the resolving of the namespace start from the root

Reply:
I think it is worth mentioning, thanks for look at this so carefully. I've updated so that it reads:

`\Sentry\init()` 

LMK if that's correct.

Reply:
Yes, it is. Also, I'm pretty sure I saw some other code snippets that should be updated as well 😉

Reply:
I found a couple of other instances and updated those.